modules/help_editor/templates/form_edit_help_content.tpl

```diff
@@ -39,6 +39,6 @@
         <input type="button" name="reset" value="Reset" class="btn btn-sm btn-primary" onclick="location.href='{$baseurl}/help_editor/edit_help_content/?section={$section}&subsection={$subsection}'" />
         <input class="btn btn-sm btn-primary" name="preview" value="Preview" type="button" />
         {/if}
-        <input class="btn btn-sm btn-primary" onclick="goBack()" value="Return to {$module_name}" type="button" />
+        <input class="btn btn-sm btn-primary" onclick="location.href='{$url}'" value="Return to {$module_name}" type="button" />
```

Comment:
I don't see the reason for `$url` to be set from a cookie and retrieved server side. Can't you just use the Referer header?

Reply:
Yeah I agree with Dave here. Let me know if you want any help.